monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/service/DataJpaTest.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.training.microservices.monolithic.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+@org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
```

Comment:
Assume, it is better to use import and put just `@DataJpaTest` on the class itself and rename the class to something like `BaseDataJpaTest` or `AbstractDataJpaTest` in order not to confuse own classes with well-known Spring’s classes.